### ISSUE TITLE
qpixel_uwsgi_1 exited with code 2

### ISSUE THREAD

**Issue by cgaz90** [codidact/qpixel#835]
**Describe the bug**
When running the qpixel docker project the uwsgi_1 container spits out the following errors pertaining to the entrypoint.sh script 

uwsgi_1  | sleep: invalid time interval ‘15\r’
uwsgi_1  | Try 'sleep --help' for more information.
uwsgi_1  | /code/docker/entrypoint.sh: line 5: $'\r': command not found
uwsgi_1  | /code/docker/entrypoint.sh: line 23: syntax error: unexpected end of file
uwsgi_1  | /code/docker/entrypoint.sh: line 2: $'\r': command not found
uwsgi_1  | sleep: invalid time interval ‘15\r’
uwsgi_1  | Try 'sleep --help' for more information.
uwsgi_1  | /code/docker/entrypoint.sh: line 5: $'\r': command not found
uwsgi_1  | /code/docker/entrypoint.sh: line 23: syntax error: unexpected end of file
qpixel_uwsgi_1 exited with code 2

Eventually the container exits with code 2 and restarts, this issue is persistent

**To Reproduce**
1. Pull from latest
2. build containers
3. run docker project/containers

**Expected behavior**
The app should run without error and be accessible on port 3000

**Desktop (please complete the following information):**
 - OS: Windows 10 - latest docker desktop

**Comment by cgaz90** [comment]
Hi, I've got it in Linux mode, its not that unfortunately

**Comment by cellio** [comment]
There's been some reworking of the Docker setup since this was filed.  @Oaphi can you review this and let us know if there's still an issue here?  (I don't have a Windows machine to test on, sorry.)

**Comment by Oaphi** [comment]
@cellio yeah, will do when I have some time to spare - I am not a big Win user these days, but still have a laptop with W10 and can dual-boot if need be. Not sure thst the Docker setup works well on Win, though, don't remember if I had a host or Docker install back then, so keeping the issue open until I can check.
